src/one_dragon/base/operation/notify_pool.py

```
@@ -0,0 +1,59 @@
+
+from cv2.typing import MatLike
+
+
+class NotifyPoolItem(NamedTuple):
+    """通知池中的一条消息"""
+    content: str
+    image: MatLike | None = None
+
+
+class NotifyPool:
+    """通知池，收集应用运行期间的节点通知消息。
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)